.github/workflows/cdn-shared-publish.yml

```diff
@@ -58,7 +61,12 @@ jobs:
           aws-region: ${{ inputs.AWS_REGION }}
 
       - name: Sync S3 content
-        run: aws s3 sync . ${{ inputs.S3URI }} --delete --exclude ".github/*" --exclude ".git/*" --exclude ".gitignore" 
+        run: |
+          if [ '${{ inputs.EXCLUDE }}' != '' ]; then
+            aws s3 sync . ${{ inputs.S3URI }} --delete --exclude ".github/*" --exclude ".git/*" --exclude ".gitignore" --exclude "${{ inputs.EXCLUDE }}"
```

Comment:
I'm curious if we can come up with a way to allow multiple values to be passed, or to allow a full override.

Instead of 
```
aws s3 sync . ${{ inputs.S3URI }} --delete --exclude ".github/*" --exclude ".git/*" --exclude ".gitignore" --exclude "${{ inputs.EXCLUDE }}""```

 maybe something like

```
aws s3 sync . ${{ inputs.S3URI }} --delete --exclude ".github/*" --exclude ".git/*" --exclude ".gitignore" "${{ inputs.ADDITIONAL_S3_SYNC_PARAMS }}"
```

and then you could pass in things like `--exclude docs --exclude another_folder --exclude yet_another_folder`.

I'm not sure it's the best implementation of that idea, but supporting "one more" folder seems like we'll run into edge cases that need "just one more" soon.

I'm okay with proceeding for now with only single folder override for now and returning to this if it ever comes up though fwiw.

Reply:
I was sorta expecting this response from you. 😁 

I started with the quick solution because I didn't want to think about how to expand a list of folders/files to skip into multiple `--exclude` cli options. I hadn't actually thought of your approach -- don't think of it as an "exclude" but just make the web dev write out the desired list of cli options they want to tack on the end.

This is an easy enough change to make. I will push an update and let you know when it's ready for review. Given my afternoon schedule (I'm out-of-office for the afternoon) it'll likely be first thing tomorrow morning.

Reply:
@JPrevost I made the change (see the WIP commit 42d07d2). I did a test run from the future-of-libraries-static repo (see the output [here](https://github.com/MITLibraries/future-of-libraries-static/actions/runs/3841012558/jobs/6540742065)) and it worked as expected.

The old `EXCLUDE` line was replaced with `SYNC_PARAMS: --exclude ".docs/*" --exclude "README.md"`.

Reply:
Looks exactly like what I was looking for. Thanks!